crypto/evp/p_lib.c

```diff
@@ -237,6 +237,17 @@ void *EVP_PKEY_get0(const EVP_PKEY *pkey)
     return pkey->pkey.ptr;
 }
 
+const unsigned char *EVP_PKEY_get_hmac(const EVP_PKEY *pkey, int *len)
```

Comment:
I don't care which function name we use. Just let me know what the consensus is and I'll change it.